pkg/networkmanager/org.cockpit-project.cockpit-networkmanager.metainfo.xml.in

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="addon">
+  <id>org.cockpit_project.cockpit_networkmanager</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <name>Networking</name>
+  <summary>Manage networking</summary>
```

Comment:
I think this should point out that it's specifically NetworkManager, especially as it's going to not work on Ubuntu (at least by default). systemd-networkd, Debian's old ifupdown, or RedHat's ifconfig are also still around.

"Cockpit configuration of NetworkManager and firewalld"?

Reply:
Should we point that out in the description?

Reply:
Yes, can't hurt. Would be nice to update the package description in tools/debian/control and tools/cockpit.spec to the same thing.

Reply:
RedHat's ifcfg is only valid for RHEL < 9 right? https://www.redhat.com/en/blog/rhel-9-networking-say-goodbye-ifcfg-files-and-hello-keyfiles

Reply:
# On RHEL kdump, networkmanager, selinux, and sosreport are part of the system package

So I'm not sure if it should really go into the system package.

Reply:
ifcfg is still being used by default on RHEL 9 cloud images. It may not be intended, but that's the status quo today. https://bugzilla.redhat.com/show_bug.cgi?id=2151504#c17 ff.

Reply:
Oh hmmm, so I should leave that out? I'm honestly not a super fan of the warning.

Reply:
What warning? I'm saying that the description should point out that this is for *NetworkManager* specifically, not just "any" networking. Hence my proposal "Cockpit configuration of NetworkManager and firewalld", but of course feel free to adjust.